libs/ui-lib/lib/common/config/docs_links.ts

```diff
@@ -98,3 +101,6 @@ export const getCiscoIntersightLink = (downloadIsoUrl: string) =>
 
 export const MTV_LINK =
   'https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.7/';
+
+export const OSC_LINK =
+  'https://www.redhat.com/en/blog/exploring-openshift-confidential-containers-solution';
```

Comment:
@jhernand  Maybe it's better to use link https://docs.redhat.com/en/documentation/openshift_sandboxed_containers which is the OSC products documentation page. Thanks.

Reply:
Done.